service-front/module/Application/view/application/pages/cp/select_address.twig

```diff
@@ -10,7 +10,7 @@
             What is the address on the ID document?
         {% endblock %}
     </h1>
-    <form method="POST" action="./select-address">
```

Comment:
This fixes the form submitting to `select-address/select-address` on remote.. It still works locally too.